userguide/system.rst

```diff
@@ -147,54 +147,55 @@ can be configured using the General tab:
    | Setting              | Value          | Description                                                                                                              |
    |                      |                |                                                                                                                          |
    +======================+================+==========================================================================================================================+
-   | Protocol             | drop-down menu | protocol to use when connecting to the administrative GUI from a browser; if modified from the default of *HTTP* to      |
-   |                      |                | *HTTPS* or to                                                                                                            |
-   |                      |                | *HTTP+HTTPS*, select the certificate to use in :guilabel:`Certificate`; if you do not have a certificate, first          |
-   |                      |                | create a CA (in `CAs`_), then the certificate itself (in `Certificates`_)                                                |
+   | Protocol             | drop-down menu | Define the web protocol to use when connecting to the administrative GUI from a browser. To change the default           |
+   |                      |                | *HTTP* to *HTTPS* or to *HTTP+HTTPS*, select a certificate to use in :guilabel:`Certificate`.                            |
+   |                      |                | If there are no certificates, first                                                                                      |
+   |                      |                | create a CA in :ref:`CAs`, then the certificate in :ref:`Certificates`.                                                  |
    |                      |                |                                                                                                                          |
```

Comment:
Not sure why "create a CA in :ref:`CAs`, then the certificate in :ref:`Certificates`." is on a new line.

Reply:
I'll fix this.

Reply:
Please do this the same way as in the other branch (with :ref:`CA <CAs>` and so on).